rasa_core/policies/ensemble.py

```diff
@@ -55,6 +57,25 @@ def _training_events_from_trackers(training_trackers):
 
         return events_metadata
 
+    def _check_priorities(self) -> None:
+        """Checks for duplicate policy priorities within PolicyEnsemble."""
+
+        priority_dict = {}
+        for p in self.policies:
+            if p.priority in priority_dict:
+                priority_dict[p.priority].append(type(p).__name__)
+            else:
+                priority_dict[p.priority] = [type(p).__name__]
+
+        for k, v in priority_dict.items():
+            if len(v) > 1:
+                logger.warn(("Found policies {} with same priority {} "
```

Comment:
(`warn` is deprecated, make sure indentation of the following lines is correct)
```suggestion
                logger.warning(("Found policies {} with same priority {} "
```